### PR TITLE
fixes for workspace switching

### PR DESCRIPTION
**Pull request by Girtablulu**
fixes towards the enabling desktop configuration

### CODE REVIEW

**Comment by Girtablulu** [comment]
#131

File: configuration/enabling-workspaces/en.md [hunk 1]
```diff
@@ -12,11 +12,14 @@ Budgie enables the switching of workspaces via the Workspace Switcher. You can e
...
+Click the Raven settings 'cog' icon {{< altimg "raven-settings-cog.jpg" "help-center/configuration/enabling-workspaces/" >}} 
 
-Click the Panel button, then in the Applets section, click the '+' button.
+Alternatively, you can open the Budgie Menu and then click "Bottom Panel" in the System Settings category.
 
-Find the Workspace Switcher entry and select it. Click the '+' button at the top of the panel. This will add the Workspace Switcher to the center section, as shown below.
+{{< altimg "budgie-panel-menu.jpg" "help-center/configuration/enabling-workspaces/" >}}
+
+
+Fine `Add applet` and click the `+` button. This will open menu to choose different applets, scroll down until you find `Workspace Switcher`, select it and click `Add Applet`
```
Comment (JoshStrobl) [inline configuration/enabling-workspaces/en.md]: - Find

File: configuration/enabling-workspaces/en.md [hunk 2]
```diff
@@ -30,17 +33,15 @@ Budgie provides a variety of keyboard shortcuts for workspaces.
...
 To view these shortcuts open the Keyboard application and scroll to the Navigation section.
 
-{{< altimg "workspace-shortcuts-gcc.jpg" "help-center/configuration/enabling-workspaces/" >}}
+{{< altimg "budgie-workspace-shortcuts-gcc.jpg" "help-center/configuration/enabling-workspaces/" >}}
 
 ### Workspaces Configuration
 
-There are 4 workspaces available in Budgie by default. If you would like to change this, you will need to install the
-GNOME Tweak Tool, available in the Software Center.
+There are 4 workspaces available in Budgie by default. For adding an new Workspace, you have to click on the `+` button.
```
Comment (JoshStrobl) [inline configuration/enabling-workspaces/en.md]: `you have to` makes it sound like it's some sort of burden, should just drop that.

**Comment by JoshStrobl** [comment]
LGTM, thanks for the fixes :+1:

**Pull request merged**